.github/workflows/dotnetcore.yml

```diff
@@ -24,16 +24,6 @@ jobs:
     - name: .NET Publish Website
       run: dotnet publish src/DevOpsMetrics.Web/DevOpsMetrics.Web.csproj --configuration Release --output ${{ github.workspace }}/web
     
-    - name: .NET Publish functional tests
```

Comment:
Please add this back. The functional tests are an important last step to ensure that everything is working as intended.